### DISCUSSION TITLE
New Shopify Express Template! OSB

### DISCUSSION THREAD

**Discussion by Michael-Gibbons** [Shopify/shopify-app-template-node#1206] [Show and tell]
Hello everyone! I've been working for the last 6 months on this open source project OSB and I finally finished v2!

[Check out the docs](https://michaelgibbons.info/OSB/#/)

https://github.com/Michael-Gibbons/OSB/tree/main

I hope it saves you all some time or teaches you something about Shopify App Development

![demo-4](https://user-images.githubusercontent.com/13829417/208221088-a4b18c72-dd38-4513-adec-4d69ee537fa3.png)
![demo-1](https://user-images.githubusercontent.com/13829417/208221089-e1191ecb-3526-4d0f-8199-e5af102d3f09.png)
![demo-2](https://user-images.githubusercontent.com/13829417/208221090-48023d6e-bc9f-4a4b-9bb5-f0231d103101.png)
![demo-3](https://user-images.githubusercontent.com/13829417/208221092-ca0e7c8d-a554-4370-883a-aa434e224a43.png)

Thank you Shopify devs for creating the express template that inspired me to build this.

**Comment by malipetek** [comment]
I was looking for something like this. I will give this a shot.

**Comment by Michael-Gibbons** [comment]
Special thanks to @paulomarg, @JaKXz, and @mkevinosullivan!
